src/DebugEngineHost/HostNatvisProject.cs

```diff
@@ -89,6 +102,20 @@ public static void FindNatvisInSolutionImpl(List<string> paths)
                 }
             }
 
+            public static string FindSolutionRootImpl()
+            {
+                string root = null;
+                string slnFile;
+                string slnUserFile;
+                var solution = (IVsSolution)Package.GetGlobalService(typeof(SVsSolution));
+                if (solution == null)
+                {
+                    return null; // failed to find a solution
+                }
+                solution.GetSolutionInfo(out root, out slnFile, out slnUserFile);
+                return root;
```

Comment:
Ah, you mean Open Folder, followed by a remote attach. I'll doubt that case works. I'll verify.